docs/reinterpreting_effects.md

```diff
@@ -181,7 +182,7 @@ handlePrint r =
     Left invalidContentTypeError -> print invalidContentTypeError
     Right ok -> case ok of
       Left jsonParseError -> print jsonParseError
-      Right facts -> traverse (putStrLn . catFact) facts
+      Right facts -> mapM_ (putStrLn . catFact) facts
```

Comment:
Good catch 👍🏻

Reply:
Super tiny nitpick, but should be `mapM_` (in the spirit of this all being `Monad`ic anyway) or should it be `traverse_` (in the spirit of the original `traverse` call)?

Reply:
Switched to `traverse_`

Reply:
I’m good either way 👍🏻